service/src/services/MeasureService.ts

```diff
@@ -183,6 +183,7 @@ export class MeasureService implements Service<fhir4.Measure> {
     // add the data requirements query params to the data requirements Library resource and add to the Library collection
     const results = { ...dataRequirements.results } as FhirLibraryWithDR;
     results['_dataRequirements'] = dataReqsQuery;
+    results.url = `Library/${dataRequirements.results.id}`;
```

Comment:
This is nitpicky but good to make sure we have consistent now I think...

When I navigate to the data requirements tab of a Measure/Library for the first time (aka it is not yet cached), then I scroll to the bottom and I see "id" that was inserted there (line 181) but I do not see the url. If I then navigate away and then back to that artifact's data requirements tab, then the url is there. In my opinion we should do one of two things:

1. Make the url "hidden", we do this with _dataRequirements by adding a projection of 0 in the `dbOperations.ts` file. I personally am leaning towards this one since the url is "made up" and I think we want to display the data requirements as a Library that follows the spec as much as possible.
2. If we want to display the url, then I would make it so that you can see the url when you first navigate to the data requirements tab. You can do this by doing `dataRequirements.results['url']...` since that is what gets returned.

Reply:
I think that this url may be accurate to the spec because it's the relative location of the data-requirements library on the server, but I'm not 100% sure. I went with option 1 for now, but if there's a reason that we think that the url might be useful to a user at some point, we can change things around to use option 2.